compiler/src/dotty/tools/dotc/typer/QuotesAndSplices.scala

```diff
@@ -288,7 +282,7 @@ trait QuotesAndSplices {
           report.error(IllegalVariableInPatternAlternative(tdef.symbol.name), tdef.srcPos)
         if variance == -1 then
           tdef.symbol.addAnnotation(Annotation(New(ref(defn.QuotedRuntimePatterns_fromAboveAnnot.typeRef)).withSpan(tdef.span)))
-        val bindingType = getBinding(tdef.symbol).symbol.typeRef
+        val bindingType = bindSymMapping(tdef.symbol).symbol.typeRef
```

Comment:
If all we use bindSymMapping for is to get the symbol of the value, why do we need to have Bind nodes as values at all instead of symbols or typerefs?

Reply:
This function returns them. But I would create them later. I will refactor this to make it simpler.